services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java

```diff
@@ -117,12 +123,53 @@ private static int getStatusCode(Throwable throwable) {
         return status;
     }
 
-    private static String getErrorCode(Throwable throwable) {
+    public static String getErrorCode(Throwable throwable) {
         if (throwable instanceof WebApplicationException && throwable.getMessage() != null) {
             return throwable.getMessage();
         }
 
-        return "unknown_error";
+        String errorMsg = "UnknownError";
```

Comment:
This change needs reverting. The `unknown_error` is a constant which must not be changed.